file.go

```diff
@@ -91,6 +91,15 @@ var (
 	msgFileCredit               = "Credit outside of cash letter"
 )
 
+// ValidateOpts contains specific overrides from the default set of validations
+type ValidateOpts struct {
+	// SkipAll will disable all validation checks of a File. It has no effect when set on records.
+	SkipAll bool `json:"skipAll"`
```

Comment:
`SkipAll` can be checked inside `File.Validate()` to bypass all checks.

Reply:
exact validation function of record will hit within reader function only
it seems that we should use skipall flag in validation function of record